JavaScript/JavaScriptSDK.Tests/CheckinTests/appInsights.tests.ts

```diff
@@ -655,6 +655,48 @@ class AppInsightsTests extends TestClass {
             }
         });
 
+        this.testCase({
```

Comment:
It feels `AppInsights._onerror` shouldn't care what the error is, only that it's dumped. I'll add these tests for `Util.dump` and make sure it handles any special cases.